// movier/dist/src/externalInterfaces/IMDBPersonNextDataInterface.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.AwardTypename = exports.ReleaseYearTypename = exports.PrimaryImageTypename = exports.NameTextTypename = exports.TextTypename = void 0;
var TextTypename;
(function (TextTypename) {
    TextTypename["Markdown"] = "Markdown";
})(TextTypename || (exports.TextTypename = TextTypename = {}));
var NameTextTypename;
(function (NameTextTypename) {
    NameTextTypename["CreditCategory"] = "CreditCategory";
    NameTextTypename["Genre"] = "Genre";
    NameTextTypename["NameText"] = "NameText";
    NameTextTypename["TitleText"] = "TitleText";
})(NameTextTypename || (exports.NameTextTypename = NameTextTypename = {}));
var PrimaryImageTypename;
(function (PrimaryImageTypename) {
    PrimaryImageTypename["Image"] = "Image";
})(PrimaryImageTypename || (exports.PrimaryImageTypename = PrimaryImageTypename = {}));
var ReleaseYearTypename;
(function (ReleaseYearTypename) {
    ReleaseYearTypename["YearRange"] = "YearRange";
})(ReleaseYearTypename || (exports.ReleaseYearTypename = ReleaseYearTypename = {}));
var AwardTypename;
(function (AwardTypename) {
    AwardTypename["AwardDetails"] = "AwardDetails";
    AwardTypename["CreditCategory"] = "CreditCategory";
    AwardTypename["NameRelationType"] = "NameRelationType";
    AwardTypename["ProductionStage"] = "ProductionStage";
})(AwardTypename || (exports.AwardTypename = AwardTypename = {}));
